### DISCUSSION TITLE
Customizing highlighting of preview window content

### DISCUSSION THREAD

**Discussion by simonmandlik** [hrsh7th/nvim-cmp#747] [Q&A] [answered]
Hi, is it possible to change highlight groups of the content in preview window? How is it specified? Thanks!

**Comment by Iamafnan** [comment]
Do you mean this ?
![IMG_20220119_223632](https://user-images.githubusercontent.com/80388154/150179475-71088c66-c8fb-4296-92df-345a6ece7254.jpg)

**Comment by simonmandlik** [comment, marked as answer]
I mean the preview floating window on the right, like in this picture

<img width="1786" alt="image" src="https://user-images.githubusercontent.com/16707112/150182905-2cbaf2c8-f511-4150-bfd5-435e4574e6bf.png">

This is some julia code highlighted with treesitter, but the colors inside the preview seems different

**Reply by smjonas** [reply]
I think the highlights in the documentation window are just the Markdown highlight groups.

**Reply by Iamafnan** [reply]
Yes it's markdown

**Reply by simonmandlik** [reply]
thanks :)
